libs/ai-endpoints/langchain_nvidia_ai_endpoints/_common.py

```diff
@@ -355,15 +355,13 @@ def get_req(
         return self._wait(response, session)
 
     def postprocess(
-        self, response: Union[str, Response], stop: Optional[Sequence[str]] = None
+        self,
+        response: Union[str, Response],
     ) -> Tuple[dict, bool]:
         """Parses a response from the AI Foundation Model Function API.
         Strongly assumes that the API will return a single response.
         """
-        msg_list = self._process_response(response)
-        msg, is_stopped = self._aggregate_msgs(msg_list)
-        msg, is_stopped = self._early_stop_msg(msg, is_stopped, stop=stop)
-        return msg, is_stopped
+        return self._aggregate_msgs(self._process_response(response))
```

Comment:
is this logic needed in _aggredate_msgs?
is_stopped = msg.get("finish_reason", "") == "stop"

Reply:
it's used in stream processing

Reply:
ok